src/global_mapper/global_mapper.h

```diff
@@ -26,25 +32,27 @@ class GlobalMapper {
   GlobalMapper(GlobalMapper&& rhs) = delete;
   GlobalMapper& operator=(GlobalMapper&& rhs) = delete;
 
-  const void IndToCoord(int ind, int ixyz[3]);
   void PushPointCloud(const PointCloud::ConstPtr& cloud_ptr);
   void Run();
 
   std::mutex cloud_mutex_;
   std::mutex map_mutex_;
+  std::mutex data_mutex_;
```

Comment:
They do need to be public.  Happy to make the change, though.  What are you guarding against or preventing by using an accessor method?